src/main/java/seedu/taassist/model/ModelManager.java

```diff
@@ -181,12 +198,14 @@ public boolean equals(Object obj) {
     }
 
     //=========== Handles focus mode state ==================================================================
+
+    // Guarantees: classToFocus must exist in TaAssist.
     @Override
     public void enterFocusMode(ModuleClass classToFocus) {
         requireNonNull(classToFocus);
-        this.focusedClass = classToFocus;
+        this.focusedClass = taAssist.findModuleClass(classToFocus).get();
```

Comment:
```suggestion
        this.focusedClass = taAssist.findModuleClass(classToFocus).orElse(null);
```
May ensure that the code will not have error if the classToFocus does not exist?

Reply:
In #112 the existence is checked through `hasModuleClass` before calling `enterFocusMode` from a command.

Is the `findModuleClass` necessary? Current `enterFocusMode` is called from two places: 
1. `ClassCommand#execute`: This already checks with `hasModuleClass` first. So the `classToFocus` is already correct. 
1. `ModelManager#setModuleClass`: Also ensured I think?

Reply:
oh alright then!